go.mod

```diff
@@ -64,6 +63,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/swaggo/files/v2 v2.0.0 // indirect
```

Comment:
This `indirect require` should be merged into the `require block` below. You can clip-paste it to below, then execute `go mod tidy`.